src/python/test/test_sklearn_rips_persistence.py

```diff
@@ -18,7 +18,8 @@ def test_rips_persistence_of_points_on_a_circle():
     NB_PC = 5
     point_clouds = [points.sphere(n_samples = random.randint(100,150), ambient_dim = 2) for _ in range(NB_PC)]
     
-    rips = RipsPersistence(homology_dimensions=[0, 1], n_jobs=-2)
+    # No edge collapse as we want to test number of H0 features with the number of points
+    rips = RipsPersistence(homology_dimensions=[0, 1], num_collapses=0, n_jobs=-2)
```

Comment:
Did the rest of the patch cause problems here, or is this independent?

Reply:
Sorry, the problem was the patch. `clear()` was removing edges. Finally, I use `prune_above_dimension(0)` instead of `clear()` to keep vertices on [4b61cbd](https://github.com/GUDHI/gudhi-devel/pull/1021/commits/4b61cbdac55e8030405033629d45d244b0cfc13b).